definitions/npm/keycode_v2.2.0/flow_v0.25.x-/keycode_v2.2.0.js

```diff
@@ -0,0 +1,290 @@
+declare module keycode {
```

Comment:
I would put the lib name into quotes, just to be consistent with all/most other definitions and Flow docs.